lib/action_controller/serialization/context.rb

```diff
@@ -0,0 +1,13 @@
+module ActionController
+  module Serialization
+    class Context
+      attr_reader :request_url, :query_parameters, :url_helpers
```

Comment:
also I usually use url helpers in serialisers a lot, so I am thinking that at some point I would like to be able to do in some serializer code something like this:
```
def validation_url
  context.validation_url(object)
end
```

Are we on the same page here? Or do you see the usage differently?